ios/FluentUI/List/FluentListSection.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// This a wrapper around `SwiftUI.Section` that has fluent style applied. It is intended to be used in conjunction with `FluentUI.FluentList` and `FluentUI.ListItem`
+/// to provide a completely fluentized list, however, it can be used on it's own if desired.
+public struct FluentListSection<SectionContent: View, SectionHeaderContent: View, SectionFooterContent: View>: View {
+
+    // MARK: Initializer
+
+    /// Creates a `FluentListSection`
+    /// - Parameters:
+    ///   - content: content to show inside of the section.
+    ///   - header: content to show inside of the header. Defaults to an `EmptyView`.
+    ///   - footer: content to show inside of the footer. Defaults to an `EmptyView`.
+    public init(@ViewBuilder content: @escaping () -> SectionContent,
+                @ViewBuilder header: @escaping () -> SectionHeaderContent = { EmptyView() },
```

Comment:
I guess we can't have nullable, but should we have other inits that only specify a header or a footer?  I did this in listItem as extensions

Reply:
Great idea. I will update this.